### PR TITLE
Oct 2022 newsletter

### PR DESCRIPTION
**Pull request by robmarcer**

### CODE REVIEW

File: src/blog/2022/11/community-news-10.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+description: News from the FlowForge and Node-RED communities
+date: 2022-11-04
+authors: ["rob-marcer"]
+---
+
+Welcome to the FlowForge newsletter for October 2022, a montly roundup of what’s happening with both FlowForge and the wider Node-RED community.
+<!--more-->
+If you've got something that you'd like us to share please email [contact@flowforge.com](mailto:contact@flowforge.com).
+
+[**FlowForge 1.0 Released**](http://localhost:8080/blog/2022/10/flowforge-1-released/)
```
Comment (ZJvandeWeg) [inline src/blog/2022/11/community-news-10.md:13]: Localhost?

File: src/blog/2022/11/community-news-10.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+[**FlowForge 1.0 Released**](http://localhost:8080/blog/2022/10/flowforge-1-released/)   
+Version 1.0 was released on 27th October. Our latest release represents our vision of the base set of features needed for you to get great value from using FlowForge in a production environment. That's not to say we are done, we will continue to add features, improve our interfaces and fix bugs with the same enthusiasm as we've worked towards 1.0. We'd like to hear your feedback on what we will be including in [1.1 and beyond](https://github.com/orgs/flowforge/projects/5). 
+
+If you’d like to learn more about what else was included in 1.0 you can do so on our [blog post](http://localhost:8080/blog/2022/10/flowforge-1-released/), on our [GitHub release page](https://github.com/flowforge/flowforge/releases/tag/v1.0.0) and on our [Youtube channel](https://www.youtube.com/watch?v=5TLT7CQR7iI). We’d also love for more of you to get involved in the development of FlowForge, [contributions to the code](https://github.com/flowforge/flowforge/blob/main/CONTRIBUTING.md) and [bug reports](https://github.com/flowforge/flowforge/issues) are really appreciated.
```
Comment (ZJvandeWeg) [inline src/blog/2022/11/community-news-10.md:16]: Localhost again

File: src/blog/2022/11/community-news-10.md (hunk 1 above)
Comment (robmarcer) [reply src/blog/2022/11/community-news-10.md:13]: Ohh, think that's a an issue with how I was running it locally, will fix.

File: src/blog/2022/11/community-news-10.md (hunk 2 above)
Comment (robmarcer) [reply src/blog/2022/11/community-news-10.md:16]: Thanks again, will fix.

File: src/blog/2022/11/community-news-10.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    
+[**FlowForge raises $7.25M to bring Node-RED to the Enterprise**](https://flowforge.com/blog/2022/10/seed-round-bring-node-red-to-enterprise/)  
+Earlier this week, we announced a $7.25M seed round led by [Cota Capital](https://www.cotacapital.com/knowledgecapital/flowforge-closes-the-gap-between-it-and-ot), joined by Westwave Capital, Uncorrelated Ventures, and Open Core Ventures. This brings a huge amount of extensive knowledge and experience in IoT, open source, enterprise-ready software solutions. You can read more about what this investment means for FlowForge in this [TechCrunch article](https://techcrunch.com/2022/11/03/flowforge-nabs-7-2m-to-help-companies-integrate-iot-using-node-red).
+
+[**FlowForge Team News**](https://flowforge.com/team/)
```
Comment (ZJvandeWeg) [inline src/blog/2022/11/community-news-10.md:21]: Would swap this item with the 3.1 release in terms of ordering.

File: src/blog/2022/11/community-news-10.md (hunk 1 above)
Comment (robmarcer) [reply src/blog/2022/11/community-news-10.md]: I've resolved this.

File: src/blog/2022/11/community-news-10.md (hunk 2 above)
Comment (robmarcer) [reply src/blog/2022/11/community-news-10.md]: I've resolved this.

**Pull request merged**